core/src/main/java/de/danoeh/antennapod/core/preferences/UserPreferences.java

```diff
@@ -797,17 +792,11 @@ public static void restartUpdateAlarm(boolean now) {
      */
     private static void restartUpdateIntervalAlarm(long triggerAtMillis, long intervalMillis) {
```

Comment:
I'd move `restartUpateIntervalAlarm()` and `restartUpdateTimeOfDayAlarm()` to `AutoUpdateManager`, and make existing `AutoUpdateManager` public methods private.

This is to make `AutoUpdateManager` encapsulates the JobService / AlarmManager implementation.